docs/modules/demos/pages/argo-cd-git-ops.adoc

```diff
@@ -0,0 +1,317 @@
+        helm:
+          releaseName: "{{ operator }}-operator"
+      destination:
+        server: "{{ server }}"
+        namespace: "stackable-operators"
+      syncPolicy:
+        syncOptions:
+          - CreateNamespace=true
+          - ServerSideApply=true
+          - RespectIgnoreDifferences=true
+        automated:
+          selfHeal: true
+          prune: true
+----
+
+The templated version for e.g. the parameters `operator=zookeeper`, `server=https://kubernetes.default.svc` and `targetRevision=25.7.0` will result in the following template:
+
+[source,yaml]
+----
+# [...]
+template:
+    metadata:
+      name: "zookeeper-operator"
+    spec:
+      project: "stackable-operators"
+      ignoreDifferences:
+        - group: "apiextensions.k8s.io"
+          kind: "CustomResourceDefinition"
+          jqPathExpressions:
+            - .spec.names.categories | select(. == [])
+            - .spec.names.shortNames | select(. == [])
+            - .spec.versions[].additionalPrinterColumns | select(. == [])
+      source:
+        repoURL: "oci.stackable.tech"
+        targetRevision: "25.7.0"
+        chart: "sdp-charts/zookeeper-operator"
+        helm:
+          releaseName: "zookeeper-operator"
+      destination:
+        server: "https://kubernetes.default.svc"
+        namespace: "stackable-operators"
+      syncPolicy:
+        syncOptions:
+          - CreateNamespace=true
+          - ServerSideApply=true
+          - RespectIgnoreDifferences=true
+        automated:
+          selfHeal: true
+          prune: true
+----
+
+This allows control over which releases and versions are deployed to which cluster.
+
+Now with ArgoCD deployed, the Sealed Secrets controller and Stackable operators up and running, you can inspect Airflow as the first Stackable product.
+
+== Airflow
+
+The Airflow web UI is reachable via Nodeport or easier, using a port-forward:
+
+[source,console]
+----
+kubectl --namespace stackable-airflow port-forward service/airflow-webserver 8080:8080
+----
+
+In your browser, go to `http://localhost:8080` and login with username `admin` and password `adminadmin`.
+
+=== Starting DAGs
+
+TODO: screenshots?
+
+== Minio / S3 - check logs
+
+Since the Airflow Kubernetes Executor will be deleted after its run, the logs are written to an S3 bucket. When accessing the logs via the Airflow webserver,
+the logs are fetched from S3 instead of the (already deleted) executor pods. The Minio / S3 instance can be accessed via port-forward:
+
+[source,console]
+----
+kubectl --namespace minio port-forward service/minio-console 9001:9001
+----
+
+Minio then is reachable via `https://localhost:9001` with username `admin` and password `adminadmin`. After the successful Airflow DAG run, logs should be stored in `demo/airflow-task-logs`.
+
+NOTE: There will be an initial warning from the Browser, that this site is insecure due to self-signed certificates. This can be ignored in this case.
```

Comment:
```suggestion
NOTE: There will be an initial warning from the browser, stating that the site is insecure due to self-signed certificates. 
This can be ignored in this case.
```